<?xml version="1.0"?>
<launch>
  <arg name="gcode_in" default="$(find gcode_translation)/gcode/15mm.gcode"/>
  <arg name="save_gcode_out" default="$(find gcode_translation)/gcode/split_register.gcode"/>
  <arg name="gcode_out" default="$(find gcode_translation)/gcode/new_15mm.gcode"/>
  <arg name="check_success" default="$(find gcode_translation)/check_success/check_success.txt"/>

  <node name="gcode_translation" pkg="gcode_translation" type="gcode_point_split" output="screen" required="true" >
    <param name="gcode_in" value="$(arg gcode_in)"/>
    <param name="save_gcode_out" value="$(arg save_gcode_out)"/>
    <param name="check_success" value="$(arg check_success)"/>
  </node>
  <include file="$(find gcode_translation)/launch/inverse_kinematics.launch">
    <arg name="gcode_out" value="$(arg gcode_out)"/>
  </include>
</launch>
